app/src/main/res/values/strings.xml

```diff
@@ -384,49 +384,49 @@
-    <string name="settings_orbot_notification_content">Failure starting Tor-as-a-Proxy. Open the Orbot app to start it manually.</string>
+    <string name="settings_orbot_notification_content">Could not start Tor-as-a-Proxy. Open the Orbot app to start it manually.</string>
 
     <string name="settings_orbot_notification_action">Open Orbot</string>
 
     <string name="settings_general_header">general</string>
 
     <string name="settings_autostart_bootup_heading">Auto-start on power-up (experimental)</string>
-    <string name="settings_autostart_bootup_desc">On device power-up, start RethinkDNS if it was running before shut down.</string>
+    <string name="settings_autostart_bootup_desc">Start up RethinkDNS with the device if it was running when it was shut off or restarted.</string>
 
-    <string name="settings_network_all_networks">Use all available networks (experimental)</string>
-    <string name="settings_network_all_networks_desc">When enabled, RethinkDNS uses all available networks to connect to Internet. If not, it uses whichever network is deemed <i>active</i> by Android.</string>
-    <string name="settings_all_networks_dialog_title">Attention!</string>
-    <string name="settings_all_networks_dialog_message">When enabled, app firewall rules for metered (ex: WiFi) and non-metered (ex: Mobile) will not be honoured.\n\nDo you want to continue?</string>
-    <string name="settings_all_networks_dialog_positive_btn">Enable anyway</string>
+    <string name="settings_network_all_networks">All networks (experimental)</string>
+    <string name="settings_network_all_networks_desc">Uses all available networks to connect to Internet insted of only the <i>active</i> one in Android.</string>
+    <string name="settings_all_networks_dialog_title">Attention</string>
+    <string name="settings_all_networks_dialog_message">Continue and ignore app firewall rules for metered (Wi-Fi) and non-metered (mobile)?</string>
+    <string name="settings_all_networks_dialog_positive_btn">Turn on anyway</string>
     <string name="settings_all_networks_dialog_negative_btn">Cancel</string>
 
-    <string name="settings_check_update_heading">Check for app updates</string>
-    <string name="settings_check_update_desc">Automatically check for app updates once a week.</string>
+    <string name="settings_check_update_heading">Look for new app versions</string>
+    <string name="settings_check_update_desc">Looks for new versions of the app once a week.</string>
 
     <string name="settings_custom_downloader_heading">Use in-app downloader</string>
-    <string name="settings_custom_downloader_desc">If disabled, Android\'s built-in <i>Download Manager</i> is used.</string>
+    <string name="settings_custom_downloader_desc">Otherwise uses Android\'s built-in <i>Download Manager</i>.</string>
 
-    <string name="settings_import_export_heading">Backup &amp; Restore</string>
+    <string name="settings_import_export_heading">Backup and Restoration</string>
```

Comment:
`Restore` is fine...? I don't think `Restoration` is something eng-speaking folks use when it comes to backups.